### ISSUE TITLE
Pointless long lines in Weitzmann 1861.

### ISSUE THREAD

**Issue by m-k94** [DCMLab/ddd#63]
For the second time now, I have noticed long lines in the text that don't make any sense.

![image](https://user-images.githubusercontent.com/75686579/113032304-0202fb00-9190-11eb-8119-8968d62cd780.png)

![image](https://user-images.githubusercontent.com/75686579/113031394-f19e5080-918e-11eb-9d7d-2af9b46b4e46.png)

I don't see how these can modify the sentence in any meaningful way. Weirdly, in both cases he accuses his critic of arguing in bad faith, but I still don't see the point of these lines. Should I transcribe them or leave them out?

**Comment by fabianmoss** [comment]
I can only see two interpretations:
Either it is meant as a pause (and thus emulating speech) before the pejorative "Nachlässigkeit" and "Ungenauigkeit", or it stands for an omitted insult such as in "was fast den Anschein einer [dummen] Nachlässigkeit hat". 

In any case, let's transcribe it as a long dash (Geviertstrich).

**Comment by fabianmoss** [comment]
On another note: Who is this Bagge and to which text does Weitzmann refer here? Maybe we should consider it, too?

**Comment by fabianmoss** [comment]
I think it is Selmar Bagge and I found this work, where he critizises Hauptmann pp. 37ff.: https://books.google.de/books?id=gylcAAAAcAAJ&pg=PP11&dq=Selmar+Bagge&hl=de&sa=X&ved=0ahUKEwjX9Or-mvTnAhWRjosKHWnSC3kQ6AEIKTAA#v=onepage&q=Selmar%20Bagge&f=false

**Comment by m-k94** [comment]
Yes, it is Selmar Bagge. As I just said during the meeting, that piece is entirely dedicated to repond to criticisms made in public newspapers. But he really puts the focus on Bagge, since he recognizes him as the most influential critic:

> Wenn ich in dem Folgenden vorzugsweise den Angriffen
> des Herrn Selmar Bagge (in der Deutschen Musikzeitung
> No. 30 und 31) begegne, so geschieht es deshalb, weil dieser
> mehr wort- als gedankenscharfe Kritiker sich zuerst der, wie er
> behauptet, „der Natur der Sache nach etwas trockenen
> Aufgabe” unterzogen hat, meine Schrift einer näheren Prüfung
> zu unterwerfen, während andere Zeitschriften sich damit be¬
> gnügten, zusammenhanglose Auszüge aus jener Recension zu
> geben, oder sich „im Ganzen ganz einverstanden mit den An¬
> sichten des Herrn S. B.” erklärten.

**Comment by m-k94** [comment]
I agree with leaving the lines in. They do feel a little bit like indications of censorship, I also had that feeling...

**Comment by m-k94** [comment]
As for the question of considering Bagge as a source: He would naturally be part of the public newspaper corpus I would like to work on. ;-)
